### PR TITLE
Update Jupyter Syntax and R & RStudio versions

### PR DESCRIPTION
**Pull request by ParadoxGuitarist**
Update Syntax

### CODE REVIEW

**Review by vwbusguy — CHANGES_REQUESTED** [review]
This is changing a whole lot more than the serverextension syntax.  I'm thinking that int-test might have gotten merged with develop at some point and that brought unintended changed into this PR.  I wonder if for now, the PR should be resubmitted against a fresh branch and we can fix or rebranch develop off master as a followup?

File: Jenkinsfile [hunk 1]
```diff
@@ -39,7 +39,6 @@ pipeline {
                         DOCKER_HUB_CREDS = credentials('DockerHubToken')
                     }
                     steps {
-                        sh 'skopeo copy containers-storage:localhost/$IMAGE_NAME docker://docker.io/ucsb/$IMAGE_NAME:latest --dest-username $DOCKER_HUB_CREDS_USR --dest-password $DOCKER_HUB_CREDS_PSW'
```
Comment (vwbusguy) [inline Jenkinsfile:42]: Why was this removed?  It should still be writing to the latest tag on success.  It doesn't on int-test, so I'm thinking this is another artifact of the int-test merger.

File: Jenkinsfile [hunk 2]
```diff
@@ -12,7 +12,7 @@ pipeline {
             stages{
                 stage('Build') {
                     steps {
-                        sh 'podman build -t localhost/$IMAGE_NAME --force-rm --pull  --no-cache .'
+                        sh 'podman build -t localhost/$IMAGE_NAME --pull --force-rm --no-cache .'
```
Comment (vwbusguy) [inline Jenkinsfile:15]: Lol, wut.

File: Containerfile [hunk 3]
```diff
@@ -1,18 +1,18 @@
-FROM jupyter/r-notebook:r-4.2.2
+FROM jupyter/r-notebook:r-4.3.1
```
Comment (vwbusguy) [inline Containerfile:1]: Sweet.  Let's update this line as well to make sure we actually ship 4.3.1 - https://github.com/UCSB-PSTAT/base-rstudio/blob/main/Jenkinsfile#L22

**Review by vwbusguy — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (ParadoxGuitarist) [reply Jenkinsfile:42]: Yeah I think you're right.

**Review by vwbusguy — APPROVED** [review]
LGTM. Ship it!  :rocket:
